en/docs/api-analytics/getting-started-guide.md

```diff
@@ -89,7 +89,7 @@ If your system connects to the service through a proxy server/firewall, you need
 
 ## Step 4 - View the Analytics Dashboards
 
-Invoke APIs and sign in to the Analytics Portal [https://analytics.choreo.dev/](https://analytics.choreo.dev/) to view the dashboards.
+Invoke APIs and open Choreo Insights [https://console.choreo.dev/insights](https://console.choreo.dev/insights) to view the dashboards.
```

Comment:
Let's make "Choreo Insights" as the link text.

Reply:
Fixed with eeb2775a30e95680c642a646120456cddef80314.